docs/tools/debugging/pro.md

```diff
@@ -0,0 +1,106 @@
+
+```sh
+tendermint debug kill <pid> </path/to/out.zip> — home=</path/to/app.d>
+```
+
+Here’s the official documentation page — <https://docs.tendermint.com/master/tools/debugging>
+
+If you’re using a process supervisor, like systemd, it will restart the Tendermint automatically. We strongly advise you to have one in production. If not, you will need to restart the node by hand.
+
+Another advantage of using Tendermint debug is that the same archive file can be given to Tendermint Core developers, in cases where you think there’s a software issue.
+
+## Tendermint debug dump
+
+Okay, but what if the node has not stalled, but its state is degrading over time? Tendermint debug dump to the rescue!
+
+```sh
+tendermint debug dump </path/to/out> — home=</path/to/app.d>
+```
+
+It won’t kill the node, but it will gather all of the above data and package it into an archive file. Plus, it will also make a heap dump, which should help if Tendermint is leaking memory.
+
+At this point, depending on how severe the degradation is, you may want to restart the process.
+
+## Tendermint Inspect
+
+What if the Tendermint node will not start up due to inconsistent consensus state? 
+
+When a node running the Tendermint consensus engine detects an inconsistent state 
+it will crash the entire Tendermint process. 
+The Tendermint consensus engine cannot be run in this inconsistent state and the so node
+will fail to start up as a result.
+The Tendermint RPC server can provide valuable information for debugging in this situation.
+The Tendermint `inspect` command will run a subset of the Tendermint RPC server 
+that is useful for debugging inconsistent state.
+
+### Running inspect
+
+Start up the `inspect` tool on the machine where Tendermint crashed using: 
+```bash
+tendermint inspect --home=</path/to/app.d>
+```
+
+`inspect` will use the data directory specified in your Tendermint configuration file.
+`inspect` will also run the RPC server at the address specified in your Tendermint configuration file.
+
+### Using inspect
+
+With the `inspect` server running, you can access RPC endpoints that are critically important
+for debugging.
+Calling the `/status`, `/consensus_state` and `/dump_consensus_state` RPC endpoint 
+will return useful information about the Tendermint consensus state.
+
+## Outro
+
+We’re hoping that these Tendermint tools will become de facto the first response for any accidents.
+
+Let us know what your experience has been so far! Have you had a chance to try `tendermint debug` or `tendermint inspect` yet?
+
+Join our [discord chat](https://discord.gg/vcExX9T), where we discuss the current issues and future improvements.
+
+—
+
+[1]: Of course, you’re free to redirect the Tendermint’s output to a file or forward it to another server.
```

Comment:
```suggestion
```

This section in particular makes this feel like a blog post to me.

Reply:
Yeah this literally was a blog post that was added to the docs (https://medium.com/tendermint/rule-76-no-excuses-debug-like-a-champion-8ef42fc52786)

Reply:
Ah, I see what happened here. I can remove most of this. I added a section to this post about how to debug using the inspect command. I'll remove the other pieces to just maintain the section on how to debug with `inspect` and we can make a separate decision on the rest of the post.